.github/workflows/build_upload_conda.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and Upload Conda Package When Release is Published
+
+# Triggers when a release is made (in our case by the run_deploy_on_merge.yml action)
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+env:
+  ANACONDA_API_TOKEN: ${{ secrets.CONDA_TOKEN }}
+  PHG2_VERSION: ${{ vars.PHG2_VERSION }}
+  PHG2_RELEASE: ${{ vars.PHG2_RELEASE }}
+  PHG2_VERSION_MD5: ${{ vars.PHG2_VERSION_MD5 }}
+
+jobs:
+  build-upload-conda:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Conda
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-update-conda: true
+          python-version: 3.11
```

Comment:
Is there a reason to use python 3.11 versus something newer?

Reply:
The newest version is 3.12 (3.13 is still prerelease) so we are pretty "bleeding edge" here. The Python version doesn't really matter all that much for this anyway, I simply included it so that it's explicit in case it breaks in the future.

Hugging Face [uses 3.8](https://github.com/huggingface/datasets/blob/main/.github/workflows/release-conda.yml#L28) in their v2 usage. If you go to the `setup-miniconda` repo their examples include a variety of versions.

Reply:
My mistake/ I was thinking this was 2.11, not 3.11.  Ignore my comment/